### PR TITLE
Relocate the `RSS user feed` button

### PR DESCRIPTION
**Pull request by devnakx**
before:
before

after:
after

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: templates/shared/user/profile_big_avatar.tmpl [hunk 1]
```diff
@@ -15,10 +15,10 @@
 		{{if .ContextUser.FullName}}<span class="header text center">{{.ContextUser.FullName}}</span>{{end}}
 		<span class="username text center">{{.ContextUser.Name}}</span>
 		{{if .EnableFeed}}
-			<a href="{{.ContextUser.HomeLink}}.rss"><i class="ui text grey gt-ml-3" data-tooltip-content="{{.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</i></a>
 		{{end}}
```
Comment (KN4CK3R) [inline templates/shared/user/profile_big_avatar.tmpl]: `{{if .EnableFeed}}` is missing.

**Review by KN4CK3R — DISMISSED** [review]

File: templates/shared/user/profile_big_avatar.tmpl (hunk 1 above)
Comment (delvh) [reply templates/shared/user/profile_big_avatar.tmpl]: Yep, I certainly missed that.

**Comment by delvh** [comment]
?

**Comment by devioa** [comment]
> ?

sorry my bad
actually, this is my first PR on github
I accidentally pressed the wrong button when I was just trying to solve this problem locally

**Comment by devioa** [comment]
alright it should be okay now

**Review by delvh — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Pull request merged**